tests/algorithms/test_bc.py

```diff
@@ -371,3 +375,59 @@ def inc_batch_cnt():
 
     # THEN
     assert batch_cnt == no_yield_after_iter
+
+
+class FloatReward(gym.RewardWrapper):
```

Comment:
why do we need this? shouldn't reward be a float already?

Reply:
Ah, I see SimpleMultiObsEnvs sometimes returns 1 rather than 1.0. This is a bug we should probably fix upstream. I'm a maintainer of SB3 so if you make a PR I can review it.

Reply:
PR [here](https://github.com/DLR-RM/stable-baselines3/pull/1676). Though the environment is more sketchy the more I look at it, e.g. they hardcode the possible transitions from each state in a way that doesn't depend on gridworld size. So maybe best to not depend on the environment much (or submit more fixes upstream).

Reply:
SimpleMultiObsEnvs is just a test env, it is not meant to be used except in tests.

Reply:
Yep, we'd only be using it in tests as well.